shared/actions/tracker.js

```diff
@@ -136,6 +141,8 @@ export function triggerIdentify (uid: string = '', userAssertion: string = ''
   }
 
   return (dispatch, getState) => new Promise((resolve, reject) => {
+    dispatch(pendingIdentify(userAssertion || uid, true))
```

Comment:
This way we know if we have an identify running for a user, and don't have to worry about limiting calls to `getProfile` or w/e.

Makes components simpler because they can blindly call `getProfile` and the right thing will happen